inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -372,6 +372,9 @@ def execute_from_manifest(self,
                 self._dispatcher_broker.send_result(SOTA_FAILURE)
                 raise SotaError(SOTA_FAILURE)
 
+    def _is_reboot_device(self) -> bool:
+        return False if self._reboot_device in ["No", "N", "n", "no", "NO"] else True
```

Comment:
**Suggestion:** The self._is_reboot_device method can be simplified by directly returning the result of the condition check, rather than using an if-else statement. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    def _is_reboot_device(self) -> bool:
        return self._reboot_device not in ["no", "n"]
```